packages/terra-content-container/src/ContentContainer.scss

```diff
@@ -1,37 +1,41 @@
-.terra-ContentContainer {
-  display: flex;
-  flex-direction: column;
+/* stylelint-disable selector-class-pattern, selector-max-compound-selectors */
```

Comment:
As new `:local {}` adds one depth to the selectors, it exceeds the max compound selectors(3).